make/compiler/Makefile.cray-prgenv

```diff
@@ -70,4 +70,13 @@ PE_CHAPEL_PKGCONFIG_LIBS := cray-pmi:cray-ugni:$(PE_CHAPEL_PKGCONFIG_LIBS)
 endif
 endif
 
+# on login/compute nodes, lustre requires the devel api to make
+# lustre/lustreapi.h available (it's implicitly available on esl nodes)
+ifeq ($(CHPL_MAKE_AUX_FILESYS), lustre)
```

Comment:
yeah, you could for instance say `CHPL_AUX_FILESYS="hdfs curl"` The [docs](http://chapel.cray.com/docs/1.12/technotes/auxIO.html#chpl-aux-filesys) mention this too.

Then in [makefile.base](https://github.com/chapel-lang/chapel/blob/master/make/Makefile.base#L95-L100) we set CHPL_MAKE_AUXFS= CHPL_MAKE_AUX_FILESYS

We also this findstring sort of thing in other places like the runtime auxio [makesfiles] (https://github.com/chapel-lang/chapel/blob/master/runtime/etc/Makefile.auxFilesys#L39-L44)